### PR TITLE
Create/doctor form

### PR DESCRIPTION
**Pull request by Hiperultimate**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -16,6 +17,9 @@ function App() {
         <Route path="/about">
           <DoctorAbout />
         </Route>
+        <Route path="/form">
```
Comment (DamFlancoz) [inline client/src/App.js:20]: Differentiate the path from patients' form, maybe use path `/doctor/form` or `/doctor-form` to . Same for `/about` path

File: client/src/components/doctorComponents/doctorForm/clinicInfo/ClinicInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+import DisplayMap from "../../../displayMap/DisplayMap.jsx";
+import DropdownList from "react-widgets/DropdownList";
+import MultiSelect from "../../../multiSelect/MultiSelect.jsx";
+
+// Have to set name for Multiselect field for forms
+function ClinicInfo({
+
+  clinicNameHook,
+  clinicAddressHook,
+  clinicConsultationFeeHook,
```
Comment (DamFlancoz) [inline client/src/components/doctorComponents/doctorForm/clinicInfo/ClinicInfo.jsx:13]: remove newline under function name

File: client/src/components/doctorComponents/doctorForm/clinicInfo/clinicInfo.css [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  grid-area: clinic-consultation-fee;
+}
+
+.input-container{
+  border-radius: 10px;
+  width: auto;
+  /* height: 38px; */
+  /* padding: 9px; */
+  background-color: var(--grey-two);
+}
```
Comment (DamFlancoz) [inline client/src/components/doctorComponents/doctorForm/clinicInfo/clinicInfo.css:92]: zombie code

File: client/src/components/mainButton/mainButton.css [hunk 4]
```diff
@@ -10,7 +10,6 @@
 }
 
 .btn-arrow{
-  /* position: absolute; */
   position: relative;
```
Comment (DamFlancoz) [inline client/src/components/mainButton/mainButton.css:13]: zombie

File: client/src/components/multiSelect/MultiSelect.jsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import "react-widgets/styles.css";
 import "./multiSelect.css";
 import Multiselect from "react-widgets/Multiselect";
 
-function MultiSelect({ options, placeholder, stateValue, handleState }) {
+function MultiSelect({ options, placeholder, stateValue, handleState}) {
```
Comment (DamFlancoz) [inline client/src/components/multiSelect/MultiSelect.jsx:5]: Add space after last parameter `function MultiSelect({ options, placeholder, stateValue, handleState }) {`

File: client/src/components/multiSelect/multiSelect.css [hunk 6]
```diff
@@ -2,6 +2,14 @@
...
-    background-color: var(--grey-two);
-} */
\ No newline at end of file
+.rw-widget-input{
+    box-shadow: none;
+    background-color: #eee;
+    border: none;
+    padding: 9px;
+    border-radius: 10px;
+}
```
Comment (DamFlancoz) [inline client/src/components/multiSelect/multiSelect.css]: Zombie

File: client/src/pages/home/Home.jsx [hunk 7]
```diff
@@ -47,10 +47,10 @@ function Home() {
   // States for Search.jsx
   const [SortOption, SwitchSortOption] = useState(false);
   const [FilterOption, SwitchFilterOption] = useState(false);
-  const [SortByVal, ChangeSortBy] = useState("");
+  const [setSortBy, ChangeSortBy] = useState("");
 
   //States for FilterSearch.jsx
-  const [locationVal, locationChange] = useState("");
+  const [setLocation, locationChange] = useState("");
   const [feeValue, changeFee] = useState([0, 500]);
```
Comment (DamFlancoz) [inline client/src/pages/home/Home.jsx:54]: When using state the first return should be a variable name like `location` and `sortBy`, don't put set in it. Make the second return name consistent like `changeSoryBy` or `sortBychange`. I will recommend sticking to `setSortby` and `setLocation`

**Review by DamFlancoz — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 1 above)
Comment (Hiperultimate) [reply client/src/App.js:20]: I think we should use conditional rendering for that. Show pages according to the type of account the user is using. If it is a doctor use doctor's page, else patient's page.

File: client/src/pages/home/Home.jsx (hunk 7 above)
Comment (Hiperultimate) [reply client/src/pages/home/Home.jsx:54]: Yes, this code was written before your last pull request comments which is why the naming is everywhere. I am using that convention everywhere now.

**Pull request merged**